src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```

Comment:
Wouldn't a talkative name be better? Something like `randomGenerator`.
Why isn't it static? Couldn't instances share it?

Reply:
thanks, I'll fix it :)